terraform/install.sh

```diff
@@ -283,16 +284,26 @@ loadGen() {
   gcloud container clusters get-credentials loadgenerator --zone "$LOADGEN_ZONE"
   kubectx loadgenerator=.
 
-  # find the IP of the load generator web interface
+  # Kicks of initial load generation via a POST request to Locust web interface
+  # This is because Locust currently doesn't support CLI and UI at the same time
   TRIES=0
-  while [[ $(curl -sL -w "%{http_code}"  "http://$loadgen_ip" -o /dev/null --max-time 1) -ne 200  && \
+#  while [[ $(curl -sL -w "%{http_code}"  "http://$loadgen_ip" -o /dev/null --max-time 1) -ne 200  && \
+  while [[ $(curl -XPOST -d "user_count=100&spawn_rate=10" http://$loadgen_ip/swarm -o /dev/null -w "%{http_code}" --max-time 1) -ne 200  && \
       "${TRIES}" -lt 20  ]]; do
     log "waiting for load generator instance..."
     sleep 10
     loadgen_ip=$(kubectl get service loadgenerator -o jsonpath='{.status.loadBalancer.ingress[0].ip}');
      [ -z "$loadgen_ip" ] && sleep 10;
     TRIES=$((TRIES + 1))
   done
+
+  # Lets load generator run for a few seconds
+  sleep 5
+
+  # Ends initial load generation
+  if [[ $(curl http://$loadgen_ip/stop -o /dev/null -w "%{http_code}") -ne 200 ]]; then
+    log "Failed to stop initial load generation"
+  fi
```

Comment:
Are you sure we want to shut down the load generation? Maybe we should keep it running so users continuously see things come in

Reply:
I'm a fan of this suggestion. This actually solves the Locust bug issue.

Reply:
Hm another Locust bug. If we start the swarm with a URL post request, users don't get the `STOP` button in the UI. 😬😬😬

The fundamental issue here is the Locust page is all jquery. I'll reduce the swarm size and increase the swarm run time by a little bit.